src/lib/runtime.js

```diff
@@ -69,7 +69,25 @@ Runtime.prototype = {
   /**
   Host instance
   */
-  hostManager: null
+  hostManager: null,
+
+  logEvent: ({eventType, task = {}, timestamp}) => {
+    if (!timestamp) {
+      timestamp = Date.now();
+    }
+
+    this.log('WORKER_METRICS', {
+      eventType,
+      worker: 'docker-worker',
+      workerPoolId: this.workerType,
+      workerId: this.workerId,
+      timestamp,
+      region: this.region,
+      instanceType: this.instanceType,
+      taskId: task.status.taskId,
+      runId: task.status.runId,
```

Comment:
These will fail when `task` is `{}`, right?

Reply:
Yes, thanks. I fixed locally!

Reply:
I came, I saw, i r+'d :)